src/docs/src/documentation/content/xdocs/zookeeperProgrammers.xml

```diff
@@ -729,7 +729,7 @@
 
       <para>Persistent recursive watches are set using the method <emphasis
       role="bold">addPersistentWatch()</emphasis>. The triggering semantics and guarantees
-      (other then one-time triggering) are the same as standard watches. Persistent recursive
+      (other than one-time triggering) are the same as standard watches. Persistent recursive
```

Comment:
(only exception regarding events is that persistent watchers **never** throw child changed event).